package.json

```diff
@@ -44,7 +44,7 @@
     "eco": "~1.1.0-rc-3",
     "eslint": "^6.7.2",
     "eslint-config-prettier": "^6.7.0",
-    "eslint-plugin-compat": "^3.3.0",
+    "eslint-plugin-compat": "^3.8.0-1",
```

Comment:
This change relies on the new feature-detection function in eslint-plugin-compat (https://github.com/amilajack/eslint-plugin-compat/pull/327), which was released in v3.8.0

Reply:
I failed to update the `package-lock.json` though. I'm getting a `npm ERR! Cannot read property 'match' of undefined` error when trying to run `npm install`. Maybe someone can help me out?